pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/PersistentTopics.java

```diff
@@ -906,11 +905,10 @@ public void expireMessagesForAllSubscriptions(@PathParam("property") String prop
 
     @POST
     @Path("/{property}/{cluster}/{namespace}/{destination}/subscription/{subName}/resetcursor/{timestamp}")
-    @ApiOperation(value = "Reset subscription to message position closest to absolute timestamp (in ms).", notes = "There should not be any active consumers on the subscription.")
+    @ApiOperation(value = "Reset subscription to message position closest to absolute timestamp (in ms).", notes = "First fence the cursor and disconnects all active consumers before resetting cursor.")
     @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
-            @ApiResponse(code = 404, message = "Topic does not exist"),
-            @ApiResponse(code = 405, message = "Not supported for global and non-persistent topics"),
-            @ApiResponse(code = 412, message = "Subscription has active consumers") })
+            @ApiResponse(code = 404, message = "Topic/Subscription does not exist"),
+            @ApiResponse(code = 405, message = "Not supported for global topics")})
```

Comment:
Already support global topics?
https://github.com/apache/incubator-pulsar/pull/79

Reply:
yes, that's correct. fixed it.

Reply:
👍